.github/workflows/Image.yaml

```diff
@@ -0,0 +1,49 @@
+name: Multi_Arch Images series build
+on: 
+  push:
+    branches:
+      - 'master'
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: docker login
+      env:
+        DOCKER_USER: ${{ secrets.DOCKER_USER }}   
+        DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }} 
+        DOCKER_BUILDKIT: 1
+        BUILDX_PLATFORMS: linux/amd64,linux/arm64,linux/ppc64le,linux/s390x,linux/386,linux/arm/v7,linux/arm/v6 
+      run: |
+        docker login -u $DOCKER_USER -p $DOCKER_PASSWORD
+    - name: install buildx
+      id: buildx
+      uses: crazy-max/ghaction-docker-buildx@v1
+      with:
+        version: latest
+
+    - name: building minimal Linux distributions with Linuxkit
+      run: |
+        docker run --rm --privileged linuxkit/binfmt:v0.8
```

Comment:
This is part of Makefile now, makefile implicitly calls when any multi arch image is build.
https://github.com/fission/fission/blob/e6c13057ceb03bdf8b052c93a855de62825f2dc0/Makefile#L46 
So we can skip this step.